data-reconciliation-app/docs/managed-ccf-demo-guidance.md

```diff
@@ -11,7 +11,7 @@ The first step is to create a Managed CCF instance. You can do this from the por
 All of the tests supplied in this sample work on all of the CCF networks. In this example you need to run `make test-mccf` to run the tests against the Managed CCF instance. This will configure the network with the correct constitution and ingest data into the network and then query the network to ensure the data is correct.
 
 ## Step 3. Update the code
-We have supplied a commented out piece of code on Line 72 of [data-schema.ts](../src/models/data-schema.ts); uncomment this.
+We have supplied a commented out piece of code on Line 80 of [data-schema.ts](../src/models/data-schema.ts#L80); uncomment this.
```

Comment:
Suggestion: 
add a snippet of the line on the docs so that in case this changes in the future people who read this doc will be able to find the line in the file.